chameleon-server/src/main/java/com/chameleonvision/common/hardware/CPU.java

```diff
@@ -0,0 +1,31 @@
+package com.chameleonvision.common.hardware;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import org.apache.commons.io.IOUtils;
+
+public enum CPU {
```

Comment:
Not sure how I feel about this whole thing. Having a class with a constructor for the 3 commands would make more sense I think.
Also, caching the values and only allowing a refresh every n milliseconds would be good too.